main.py

```diff
@@ -1,9 +1,12 @@
-from flask import Flask, jsonify, send_file
+from flask import Flask, jsonify, send_file, request, g
```

Comment:
looks like g is unused so you can remove it

Reply:
I added it due to a problem. So in the POST, if you add a recipe it doesn't commit to the db. I've tried well everything i know. Once you rerun, it disappears.

Reply:
I added it due to a problem. So in the POST, if you add a recipe it doesn't commit to the db. I've tried well everything i know. Once you rerun, it disappears.